apps/src/code-studio/pd/workshop_dashboard/AttendancePanel.jsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import {Row, Col, Button} from 'react-bootstrap';
+import WorkshopPanel from './WorkshopPanel';
+import {DATE_FORMAT} from './workshopConstants';
+
+/**
+ * Provides controls for taking attendance in a workshop session.
+ */
+export default class AttendancePanel extends React.Component {
```

Comment:
I see that we change file name style, for example AttendancePanel.jsx (new) vs. workshop_enrollment.jsx (old). What is the convention to follow?

Reply:
I'd like us to move to `PascalCase` instead of `snake_case` for JavaScript filenames, as this is the convention other cabals are using.